From 0b92c90b1c170d078ea60dec17ea5a99e18be90d Mon Sep 17 00:00:00 2001
From: Di Wu <diwux@google.com>
Date: Sat, 11 Apr 2026 00:00:00 -0700
Subject: [PATCH] Revert "Reland "Remove base/logging.h include from validation_errors.h""

This reverts commit 0b92c90b1c170d078ea60dec17ea5a99e18be90d to fix widespread
IWYU breakages downstream in ChromeOS.
---
 mojo/public/cpp/bindings/lib/validation_errors.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/mojo/public/cpp/bindings/lib/validation_errors.h b/mojo/public/cpp/bindings/lib/validation_errors.h
index 2e1437787e..2ca3107907 100644
--- a/mojo/public/cpp/bindings/lib/validation_errors.h
+++ b/mojo/public/cpp/bindings/lib/validation_errors.h
@@ -10,6 +10,7 @@
 #include "base/component_export.h"
 #include "base/dcheck_is_on.h"
 #include "base/functional/callback_forward.h"
+#include "base/logging.h"
 #include "mojo/public/cpp/bindings/lib/send_validation_type.h"
 
 namespace mojo {
-- 
2.43.0
